repos/system_upgrade/el7toel8/actors/checkfstaboptions/tests/test_checkfstaboptions.py

```diff
@@ -0,0 +1,89 @@
+from leapp.models import FstabEntry, StorageInfo
+from leapp.snactor.fixture import current_actor_context
+from leapp.reporting import Report
+
+
+MOUNTS_CORRECT_1 = [
```

Comment:
watch the previous comment about constants

Reply:
maybe to add that tuples are usually more memory efficient and faster (https://stackoverflow.com/questions/1708510/list-vs-tuple-when-to-use-each)

Reply:
Thanks, it's a tuple now.